lib-extra/build.gradle

```diff
@@ -16,6 +16,8 @@ dependencies {
 	implementation "com.googlecode.concurrent-trees:concurrent-trees:2.6.1"
 	// used for xml parsing in EclipseFormatter
 	implementation "org.codehaus.groovy:groovy-xml:3.0.9"
+	// used for pom sorting
+	implementation 'com.github.ekryd.sortpom:sortpom-sorter:3.0.0'
```

Comment:
This dep needs to be resolved like this:

https://github.com/diffplug/spotless/blob/85344bdbe94bac86a0723910dc6d32979d566bcc/lib/src/main/java/com/diffplug/spotless/java/GoogleJavaFormatStep.java#L122